cli/auth.go

```diff
@@ -359,8 +359,8 @@ func refreshToken(ctx context.Context, httpClient *http.Client, token *Token) (*
 	return buildToken(resp, token.TokenURL, token.ClientID)
 }
 
-func processTokenResposne(res *http.Response) (*tokenResponse, error) {
-	defer utils.UncheckedErrorFunc(res.Body.Close)
+func processTokenResponse(res *http.Response) (*tokenResponse, error) {
+	defer utils.UncheckedError(res.Body.Close())
```

Comment:
You'll need to put this into a defer func() { that calls the non Func version of unchecked. Defer arguments are evaluated when defer statement is first passed over. As it's currently written, the code will immediately close the response

Reply:
*facepalm* I swear I knew that, and that's why I preferred the Func variant of the utility... Just had a brain fart and forgot I was swapping the two in the unsafe direction.

Reply:
Noting for posterity, but the linter loses track of it when it's deferred in an anonymous func(), so reverting to the original UncheckedErrorFunc and adding nolint statements.